generators/cypress/index.js

```diff
@@ -108,4 +137,57 @@ module.exports = class extends BaseBlueprintGenerator {
     if (useBlueprints) return;
     return this._writing();
   }
+
+  _postWriting() {
+    return {
+      configureCoverage() {
+        if (!this.cypressCoverage) return;
+        this.packageJson.merge({
+          devDependencies: {
+            'babel-loader': this.configOptions.dependabotPackageJson.devDependencies['babel-loader'],
+            'babel-plugin-istanbul': this.configOptions.dependabotPackageJson.devDependencies['babel-plugin-istanbul'],
+            '@cypress/code-coverage': this.configOptions.dependabotPackageJson.devDependencies['@cypress/code-coverage'],
+          },
+          scripts: {
+            'prewebapp:instrumenter': 'npm run clean-www && rimraf .nyc_output coverage',
+            'webapp:instrumenter': 'ng build --configuration instrumenter',
```

Comment:
```suggestion
            'clean-coverage': 'rimraf .nyc_output coverage',
            'pree2e:cypress:coverage': 'npm run clean coverage && npm run ci:server:await',
            'e2e:cypress:coverage': 'npm run e2e:cypress',
            'poste2e:cypress:coverage': 'nyc report',
            'prewebapp:instrumenter': 'npm run clean-www && npm run clean-coverage',
            'webapp:instrumenter': 'ng build --configuration instrumenter',
```
Add script to print report.